.github/workflows/test.yml

```diff
@@ -3,56 +3,50 @@ on:
   push:
     branches: ["main"]
   pull_request_target:
```

Comment:
I think this should be changed to `pull_request`

Reply:
No, because then we'd run the action from the pull request not from main.

Reply:
so both actions will use pull_request_target